aeimer/cachetool-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+    "bundles": {
+        "CacheTool\\Bundle\\CacheToolBundle": ["all"]
+    },
+    "container": {
+        "env(CACHETOOL_ADAPTER)": "fastcgi",
+        "env(CACHETOOL_FASTCGI)": "127.0.0.1:9000",
+        "env(CACHETOOL_APC)": "true",
+        "env(CACHETOOL_APCU)": "true",
+        "env(CACHETOOL_OPCACHE)": "true"
```

Comment:
Have a look here how to configure environment variables?

https://github.com/symfony/recipes#env-configurator

Reply:
I want to set some defaults. Thats the way I've learned it... Waht do you suggest?